### PR TITLE
[Several products Marcio] Replace click select

### PR DESCRIPTION
**Pull request by marciocloudflare**
- Replaces instances where "click" appears with "select" in:
- 1dot
- Web Analytics
- Email Routing
- Fundamentals > Notifications
- Images
- Railgun
- Registrar
- Warp Client
- Zaraz
- Addresses PCX-4380

### CODE REVIEW

File: content/analytics/web-analytics/getting-started/_index.md [hunk 1]
```diff
@@ -12,30 +12,30 @@ meta:
...
 
 1. Go to [Get started with Web Analytics](https://dash.cloudflare.com/sign-up/web-analytics) to log in to your Cloudflare account. You can also use this link to create one if you need to.
-2. Click **Add a site**.
+2. Select **Add a site**.
 3. In **Set up hostname**, write your website's hostname.
-4. Click the message box that appears to choose the hostname you have input and click **Done**.
+4. Select the message box that appears to choose the hostname you have input and select **Done**.
 5. Copy the JS snippet from **Manage site**. This is also where you can later edit the hostname you have just added.
-6. (Optional) **Click View Analytics sites** to go back on the Web Analytics interface. If you prefer to continue setting up Web Analytics website, continue reading.
+6. (Optional) select **View Analytics sites** to go back on the Web Analytics interface. If you prefer to continue setting up Web Analytics website, continue reading.
```
Comment (angelampcosta) [inline content/analytics/web-analytics/getting-started/_index.md]:
```suggestion
6. (Optional) Select **View Analytics sites** to go back on the Web Analytics interface. If you prefer to continue setting up Web Analytics website, continue reading.
```

File: content/fundamentals/notifications/create-notifications/create-pagerduty.md [hunk 2]
```diff
@@ -27,10 +27,10 @@ According to PagerDuty, you need an account with the following permissions to ad
...
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account.
 2. Go to **Notifications**.
-3. Click **Destinations** on the left side of your dashboard.
-4. In the **Connected notification services** card, click **Connect**.
+3. Select **Destinations** on the left side of your dashboard.
+4. In the **Connected notification services** card, select **Connect**.
 5. Log in to your PagerDuty account to connect it to your Cloudflare account.
-6. Select the services you want to use and click **Connect**.
-7. The browser will navigate back to your Cloudflare dashboard. Click **Continue**.
+6. Select the services you want to use and select **Connect**.
```
Comment (angelampcosta) [inline content/fundamentals/notifications/create-notifications/create-pagerduty.md]: Could we say "Choose the services... ", so you don't repeat select in the same sentence.

File: content/fundamentals/notifications/edit-notifications/edit-pagerduty.md [hunk 3]
```diff
@@ -8,8 +8,8 @@ title: Edit a PagerDuty connected service
 To edit which PagerDuty services are connected to your Cloudflare account, you first have to disconnect PagerDuty from Cloudflare, make any changes you need in PagerDuty, and then reconnect it. Be aware that disconnecting PagerDuty will disable notifications being sent to PagerDuty where currently configured. If other delivery destinations (like email) were selected, then those notifications will still be routed as configured. If PagerDuty was the only configured destination, disconnecting PagerDuty may result in a notification with no destination.
 
 1. Sign in to your [Cloudflare dashboard](https://dash.cloudflare.com/login).
-2. Go to **Notifications** and click **Destinations** on the left side of your dashboard.
-3. In the **Connected notification services** card, click **View** on the PagerDuty service you want to disconnect.
-4. Click **Disconnect** and select **Confirm**.
+2. Go to **Notifications** and select **Destinations** on the left side of your dashboard.
+3. In the **Connected notification services** card, select **View** on the PagerDuty service you want to disconnect.
+4. Select **Disconnect** and select **Confirm**.
```
Comment (angelampcosta) [inline content/fundamentals/notifications/edit-notifications/edit-pagerduty.md]:
```suggestion
4. Select **Disconnect** > **Confirm**.
```

File: content/images/cloudflare-images/transform/resize-images.md [hunk 4]
```diff
@@ -38,13 +38,13 @@ Variants allow you to choose what to do with your image's metadata information,
...
 
-Clicking the **Always allow public access** option will make a particular variant always publicly accessible, even when [making images private](/images/cloudflare-images/make-an-image-private/). 
+Selecting the **Always allow public access** option will make a particular variant always publicly accessible, even when [making images private](/images/cloudflare-images/make-an-image-private/). 
 
 ## How to resize an image
 
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account.
-2. Click **Images** > **Variants**.
-3. Name your variant and click **Add New Variant**.
+2. CliSelectck **Images** > **Variants**.
```
Comment (angelampcosta) [inline content/images/cloudflare-images/transform/resize-images.md]:
```suggestion
2. Select **Images** > **Variants**.
```

**Review by angelampcosta — APPROVED** [review]

File: content/fundamentals/notifications/create-notifications/create-pagerduty.md (hunk 2 above)
Comment (marciocloudflare) [reply content/fundamentals/notifications/create-notifications/create-pagerduty.md]: Yes. Thanks

**Pull request merged**